src/features/projectsV2/ProjectSnippet/microComponents/ProjectBadge.tsx

```diff
@@ -3,7 +3,7 @@ import OffSiteReviewedIcon from '../../../../../public/assets/images/icons/proje
 import FieldReviewedIcon from '../../../../../public/assets/images/icons/projectV2/FieldReviewedIcon';
 import TopProjectIcon from '../../../../../public/assets/images/icons/projectV2/TopProjectIcon';
 import NewInfoIcon from '../../../../../public/assets/images/icons/projectV2/NewInfoIcon';
-import style from '../../styles/Badge.module.scss';
+import style from '../styles/Badge.module.scss';
 import { useTranslations } from 'next-intl';
```

Comment:
This is a personal preference - while importing styles, I like to name this as `styles` instead of `style` as it makes it easier to search.

`style` can refer to the `style` property used for inline styles within html.